app/components/support_interface/reference_with_feedback_component.rb

```diff
@@ -156,9 +152,19 @@ def email_address_row
     end
 
     def relationship_row
+      value = tag.p(relationship, class: 'govuk-body')
+
+      if reference.feedback_provided?
+        if relationship_correction.present?
+          value += tag.p("#{reference.name} said:", class: 'govuk-body') + tag.p(relationship_correction, class: 'govuk-body')
+        else
+          value += tag.p("This was confirmed by #{reference.name}", class: 'govuk-body')
+        end
+      end
+
       row = {
-        key: 'Relationship to candidate',
-        value: relationship,
+        key: 'How the candidate knows them and how long for',
+        value: value,
```

Comment:
Can we add tests for this method?

Reply:
Good idea. Done in [52c976a](https://github.com/DFE-Digital/apply-for-teacher-training/pull/7620/commits/52c976a1fc9e86daeca93e045b988edabb6f9bc6).